# done
start:
  pose:
    position: {x: 7.24826894463, y: 4.04479364819, z: 0.0}
    orientation: { x: 0.0, y: 0.0, z: -0.91626811922, w: 0.400565517365}

# done
finish:
  pose:
    position: { x: 6.49953619164, y: -0.733265312455, z: 0.0}
    orientation: {x: 0.0, y: 0.0, z: 0.971973919883, w: 0.23508870468}

# done
middle:
  pose:
    position: { x: 6.28203298434, y: 1.45009345075, z: 0.0}
    orientation: {x: 0.0, y: 0.0, z: -0.999269840631, w: 0.0382071407751}

# done might
wait:
  pose:
    position: {  x: 5, y: 1.69, z: 0.00247}
    orientation: { x: 0.0, y: 0.0, z: 0.990859995239, w: 0.134894291334}

# done might
wait_centre:
  pose:
    position: { x: 2.97595373904, y: 2.05197043334, z: 0.0}
    orientation: { x: 0.0, y: 0.0,   z: 0.993882592579, w: 0.110441804442}


# done
lift_centre:
  pose:
    position: { x: 1.25456318233, y: 2.41576626914, z: 0.0}
    orientation: { x: 0.0, y: 0.0, z: -0.0886847644742, w: 0.996059743464}


# done
wait_position_center_point: [3.37838471113, 1.99490843094 ]

# done
#wait_position_points:
#  [[ 2.0859310626983643, -3.3872008323669434 ],
#   [ 4.024967670440674,  -3.3741872310638428 ],
#   [ 4.011953830718994,  -5.768733024597168  ],
#   [ 1.7605891227722168, -5.794760704040527  ] ]

# from clicked
wait_position_points:
  [[  3.17698693275, 3.43827986717 ],
   [ 5.88484573364, 2.86555981636 ],
   [ 5.78069639206, 0.887074172497 ],
   [ 2.53907728195, 0.756910562515  ] ]


# done
lift_position_center_point: [  1.25456318233, 2.41576626914 ]

# done
lift_position_points:
  [[ 2.07351517677, 2.80156946182  ],
   [ 0.796780288219, 2.97078204155 ],
   [ 0.692557096481, 1.81232655048 ],
   [ 1.87809658051, 1.68216264248  ]]

# done
lift:
  pose:
    position:
      x: 1.25456318233
      y: 2.41576626914
      z: 0.0
    orientation:
      x: 0.0
      y: 0.0
      z: -0.0886847644742
      w: 0.996059743464

# done
floor:
  status: "unknown"
  number: 2

# done
counter_door:
  counter: 0

# done
in_lift:
  status: False


# done
counter_lift:
  counter: 0

# done
door:
  pose:
    position: { x: 2.25366711617, y: 2.26118898392, z: 0.0 }
    orientation: { x: 0.0, y: 0.0, z: 0.999978960672, w: 0.0064867722379}

# done not well
lift_in_front:
  pose:
    position: {  x: 3.2, y: 2.37, z: 0.00256}
    orientation: {  x: 0.0, y: 0.0, z: 0.990859995239, w: 0.134894291334 }



# done
counter_door_open:
  counter: 0

# done
door_location: [ 2.25366711617, 2.26118898392]

#door_location: [ 2.16524496803, 2.25163928304]


# done
test_lift_points:
  - [ 2.07351517677, 2.80156946182  ]
  - [ 0.796780288219, 2.97078204155 ]
  - [ 0.692557096481, 1.81232655048 ]
  - [ 1.87809658051, 1.68216264248  ]

# done
corners_arena:
   [[ 1.96626102924, -0.987280905247 ],
   [ 7.12160825729, -1.5339679718],
   [ 7.83762931824, 4.70086622238],
   [ 3.28113484383, 5.32565116882]]

# done
phase3_lift:
  pose:
    position: {  x: 3.04619555256, y: 2.05055486019, z: 0.0}
    orientation: { x: 0.0, y: 0.0, z: -0.0690247424816, w: 0.997614948227 }


#done
door_detected_official: True
